utilities/exceptions.py

```diff
@@ -23,21 +23,21 @@ class InferenceResponseError(Exception):
     pass
 
 
-class InvalidStorageArgument(Exception):
+class InvalidStorageArgumentError(Exception):
     def __init__(
         self,
-        storageUri: Optional[str],
+        storage_uri: Optional[str],
         storage_key: Optional[str],
         storage_path: Optional[str],
     ):
-        self.storageUri = storageUri
+        self.storageUri = storage_uri
         self.storage_key = storage_key
         self.storage_path = storage_path
 
     def __str__(self) -> str:
         msg = f"""
             You've passed the following parameters:
-            "storageUri": {self.storageUri}
+            "storage_uri": {self.storageUri}
             "storage_key": {self.storage_key}
             "storage_path: {self.storage_path}
             In order to create a valid ISVC you need to specify either a storageUri value
```

Comment:
storageUri to storage_uri

Reply:
because python vars should be lower case

Reply:
I meant you should also update the name in the docstring at L43, sorry

Reply:
done